### PR TITLE
Add ssl config option to three tier app and use in tests

### PR DESCRIPTION
**Pull request by duncangrant**

### CODE REVIEW

File: catalog.tests.bom [hunk 1]
```diff
@@ -27,6 +27,12 @@ brooklyn.catalog:
       services:
       - type: ThreeTierApp
         id: target-app
+        brooklyn.config:
+          ssl:
+            certificateSourceUrl:
+              "https://raw.githubusercontent.com/brooklyncentral/brooklyn-three-tier-webapp/add-ssl/resources/domain.crt"
```
Comment (m4rkmckenna) [inline catalog.tests.bom]: This does not exist

Comment (drigodwin) [inline catalog.tests.bom]: You need a branch in the URL such as `/master/`

**Review by drigodwin — CHANGES_REQUESTED** [review]

File: catalog.tests.bom (hunk 1 above)
Comment (duncangrant) [reply catalog.tests.bom]: so why does https://raw.githubusercontent.com/duncangrant/brooklyn-three-tier-webapp/add-ssl/resources/domain.key exist?

Comment (duncangrant) [reply catalog.tests.bom]: crap - because I copied the wrong branch name

Comment (drigodwin) [reply catalog.tests.bom]: `/add-ssl/` is the branch - that doesn't exist in this repo.

**Review by drigodwin — APPROVED** [review]
Looks good and the tests pass for me, thanks @duncangrant

**Pull request merged**